tests/numpy/test_stack.py

```diff
@@ -0,0 +1,98 @@
+# coding=utf-8
+# Copyright (C) 2020 NumS Development Team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import numpy as np
+
+
+# pylint: disable=import-outside-toplevel
+
+def try_multiple_nd(n):
```

Comment:
This looks good, and we can keep it as-is, but would be great to improve coverage by doing something like this:

```python
import itertools

python_types = [
1, 
[1, 2], 
[[1,2]],
[[[1,2]]]
]
all_types = python_types + list(map(np.array, all_types))
test_cases = list(itertools.product(all_types, repeat=3))

for arguments in test_cases:
   if n == 1:
      np_res = np.atleast_1d(*arguments)
      nps_res = nps.atleast_1d(*arguments)
      ...
```

This should cover all the numpy cases, and I think it's okay given the small number of cases it ends up generating. It would also be good to mix in a few random cases with BlockArray instances for the `nps` implementations, e.g. by drawing from a set like this:

```python
ba_cases = list(map(nps.array, all_types))
```

Reply:
```all_types = python_types + list(map(np.array, all_types))```

Is the second parameter of `map` correct? How can it use `all_types` before it's created?

Reply:
Yep you're right, the input to map should be `python_types`.